oauth/authcode.go

```diff
@@ -234,8 +243,12 @@ func (ac *AuthorizationCodeTokenSource) Token() (*oauth2.Token, error) {
 		c: codeChan,
 	}
 
+	// strip protocol prefix from configured redirect url for local webserver
```

Comment:
I'm noodling on whether it would be better to do something like
```
u, _ := url.Parse(ac.getRedirectUrl())
redirectServer := fmt.Sprintf("%s:%d", u.Hostname(), u.Port())
```

Any thoughts?

Reply:
That's probably better than the trim calls. I do wonder though if we should be using the configured hostname or always use `localhost` instead... like what if the hostname doesn't resolve to localhost? 🤔 I suppose it doesn't matter for our use-case so could always be modified later.